Dockerfile

```
@@ -0,0 +1,31 @@
+FROM debian/eol:wheezy
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+     software-properties-common \
+     bison \
+     flex  \
+     texinfo  \
+     gcc \
+     g++ \
+     make \
+     curl \
+     sudo \
+     ca-certificates \
+     file \
+     autoconf \
+     libz-dev \
+     gettext \
+     libcurl4-openssl-dev \
+     wget
```

```suggestion
     libcurl4-openssl-dev
```
`wget` is not needed.